sparse_strips/vello_sparse_tests/tests/issues.rs

```diff
@@ -340,3 +342,30 @@ fn do_not_panic_on_multiple_flushes(ctx: &mut impl Renderer) {
     ctx.flush();
     ctx.fill_rect(&Rect::new(0.0, 0.0, 4.0, 4.0));
 }
+
+// See <https://github.com/linebender/vello/issues/1129>
+#[vello_test]
+fn gradient_with_last_stop_0_opacity(ctx: &mut impl Renderer) {
```

Comment:
I think if you pull from main, you’ll see another test called `gradient_color_alpha` added in #1142